speech_to_text.py

```diff
@@ -319,7 +329,22 @@ def receive_done_job() -> Optional[dict]:
     elif len(messages) == 0:
         return None
     else:
-        raise Exception("received more than one message from todo queue!")
+        raise SpeechToTextException("received more than one message from todo queue!")
+
+
+def inspect_media(path) -> dict:
+    try:
+        output = subprocess.check_output(
+            ["ffprobe", "-show_format", "-print_format", "json", "-v", "quiet", path]
+        )
+        result = json.loads(output)
+        return {
+            "duration": float(result["format"]["duration"]),
+            "format": result["format"]["format_name"],
+            "size": int(result["format"]["size"]),
+        }
```

Comment:
I extracted just the things we wanted just in case we ever want to use something other than ffprobe to inspect the media.